beyond-essence/inventory/radio-buttons.md

```diff
@@ -0,0 +1,58 @@
+---
+layout: beyond-essence
+header_image: private/main.jpg
+title: Radio buttons
+background_image: index.jpg
+---
+
+Radio buttons belong to the selection controls and allow users to select a single option from a list.
+
+## Use cases
+
+In the Beyond cockpit we provide two types of radio buttons:
+
+* Radio buttons that relate to input fields
+
+{% image_custom image="/assets/img/pages/essence/radio-buttons-input.png" width="100" lightbox %}
+
+* Radio buttons that open content
+
+{% image_custom image="/assets/img/pages/essence/radio-buttons-open.gif" width="100" lightbox %}
+
```

Comment:
We also have a use case with "only" radio buttons.
<img width="989" alt="Bildschirmfoto 2019-08-13 um 12 58 00" src="https://user-images.githubusercontent.com/6104802/62937725-bf7b8780-bdcd-11e9-8e5e-a77762dbd40e.png">